health/micro-ui/web/micro-ui-internals/packages/modules/microplan/src/components/UserAccess.js

```
@@ -149,7 +156,7 @@ function UserAccess({ category }) {
           className={"roleComposer"}
           type={"default"}
           heading={t(`${category}`)}
-          children={[<RoleTableComposer />]}
+          children={[<RoleTableComposer category={category} nationalRoles={nationalRoles} />]}
```

Biome

[error] Avoid passing children using a prop

The canonical way to pass children in React is to use JSX elements

(lint/correctness/noChildrenProp)